transform/mappings/currency.py

```diff
@@ -0,0 +1,7 @@
+"""Mapping of currency symbols to their corresponding three-letter acronyms."""
+
+currency_mapping = {
```

Comment:
if we have only 3 currency symbols its fine. If there could be more - I would suggest using some 3party package, 
e.g. https://pypi.org/project/pycountry/ - includes all info about countries. 
OR something more lightweight:
https://pypi.org/project/currency-symbols/

Reply:
None of the third-party libraries available, such as forex_python, py-country, or currency-symbols, support the mapping of currency symbols to 3-digit codes. This is because the $ symbol is used by 26 different currencies.